dev/dependencyList

```diff
@@ -108,6 +112,7 @@ simpleclient_tracer_otel/0.14.1//simpleclient_tracer_otel-0.14.1.jar
 simpleclient_tracer_otel_agent/0.14.1//simpleclient_tracer_otel_agent-0.14.1.jar
 slf4j-api/1.7.35//slf4j-api-1.7.35.jar
 snakeyaml/1.28//snakeyaml-1.28.jar
+sqltool/2.5.2//sqltool-2.5.2.jar
```

Comment:
The license is "HSQLDB License, a BSD open source license", I'm not sure if it accepted by Apache, have you see other Apache projects bundle it?